website/static/js/fangorn.js

```diff
@@ -2230,6 +2259,23 @@ tbOptions = {
     uploads : true,         // Turns dropzone on/off.
     columnTitles : _fangornColumnTitles,
     resolveRows : _fangornResolveRows,
+    // Move obj.data.attributes contents to obj.data , for WB v1 API
+    lazyLoadPreprocess: function(obj) {
+        function moveAttrs(obj) {
```

Comment:
Important: you have two variables called `obj` in this scope.  Please rename on of them.